docs/how-to-docs/anti-affinity.md

```diff
@@ -0,0 +1,45 @@
+## Affinity/Anti-affinity
+
+###  Overview
+In Kubernetes you specify an affinity (or anti-affinity) of a pod relative to a group of pods it can be placed with. The node does not have control over the placement. The affinity feature consists of two types, "affinity" and "anti-affinity". 
+
+Pod affinity can tell the scheduler to locate a new pod on the same node as other pods if the label selector on the new pod matches the label on the current pod.
+
+Pod anti-affinity can prevent the scheduler from locating a new pod on the same node as pods with the same labels if the label selector on the new pod matches the label on the current pod.
+
+There are two types of pod affinity rules: required and preferred. Required rules must be met before a pod can be scheduled on a node. You can think of them as "hard" and "soft" respectively, in the sense that the former specifies rules that must be met for a pod to be scheduled onto a node, while the latter specifies preferences that the scheduler will try to enforce but will not guarantee. Anti-affinity is specified as field nodeAffinity of field affinity in the PodSpec.
+
+| Required | Preferred
+| ----------- | ----------- |
+| `requiredDuringSchedulingIgnoredDuringExecution` | `preferredDuringSchedulingIgnoredDuringExecution` | 
+| Specifies rules that must be met for a pod to be scheduled onto a node | Specifies preferences that the scheduler will try to enforce | 
+| Example: "co-locate the pods of service A and service B in the same zone, since they communicate a lot with each other" | Example: "spread the pods from this service across zones" (a hard requirement wouldn't make sense, since you probably have more pods than zones). | 
+
```

Comment:
Mention that preferred rules are safer because if a required rule cannot be met, pods won't get scheduled.